src/Symfony/Component/Console/Tests/Command/CompleteCommandTest.php

```diff
@@ -125,6 +125,57 @@ public function provideCompleteCommandInputDefinitionInputs()
         yield 'custom-aliased' => [['bin/console', 'ahoy'], ['Fabien', 'Robin', 'Wouter']];
     }
 
+    /**
+     * @dataProvider provideZshCompleteCommandNameInputs
+     */
+    public function testZshCompleteCommandName(array $input, array $suggestions)
```

Comment:
I'm not sure we need this specific tests here. I'll add at least `ZshCompletionOutputTest`.